src/SmartComponents/BaselinesPage/CreateBaselineModal/CreateBaselineModal.js

```diff
@@ -28,21 +29,34 @@ class CreateBaselineModal extends Component {
 
         this.handleChecked = (_, event) => {
             const { value } = event.currentTarget;
-            this.props.clearSelectedBaselines();
+            let fromScratchChecked = false;
+            let copyBaselineChecked = false;
+            let copySystemChecked = false;
 
             if (value === 'fromScratchChecked') {
-                this.setState({ fromScratchChecked: true, copyBaselineChecked: false, copySystemChecked: false });
+                fromScratchChecked = true;
             } else if (value === 'copyBaselineChecked') {
-                this.setState({ fromScratchChecked: false, copyBaselineChecked: true, copySystemChecked: false });
+                copyBaselineChecked = true;
             } else {
-                this.setState({ fromScratchChecked: false, copyBaselineChecked: false, copySystemChecked: true });
+                copySystemChecked = true;
             }
+
+            this.setState({
+                fromScratchChecked,
+                copyBaselineChecked,
+                copySystemChecked,
+                selectedBaseline: ''
+            });
         };
     }
 
+    isBaselineSelected(baselineId) {
+        this.setState({ selectedBaseline: baselineId });
```

Comment:
The action of this function doesn't fit with my expectation of it's name. The `is` portion of the function name makes me think this will return whether a baseline is selected or not rather than selecting a baseline.